project/plugins.sbt

```diff
@@ -1,6 +1,12 @@
+enablePlugins(BuildInfoPlugin)
+
+val sbtProtocV = "0.99.25"
```

Comment:
Good to make those consistent!

I think I've indeed seen that 'listbuffer casting' issue indeed, though I'm not sure where - would indeed be great to fix that.

I'm not sure about he downgrade from 0.99.26 to 0.99.25: while it's only a patchversion, there's a number of commits in there, including the https://github.com/thesamet/sbt-protoc/commit/ac1cfa086577f0e5a9a1bf5f5a14d464a7c550a8 revert by @bjaglin, and I'm not sure if downgrading would lead to trouble here...

Not quite sure about the downgrade though -

Reply:
Sure, a fix is the final intention. Let's keep it open. I will try to find a solution that is last hacky and more stable.